TotalSegmentator/TotalSegmentator.py

```diff
@@ -689,10 +689,11 @@ def pipInstallSelective(self, packageToInstall, installCommand, packagesToSkip):
 
             match = False
             if not match:
-                # ruff ; extra == 'dev' -> rewrite to: ruff[extra]
-                match = re.match(r"([\S]+)[\s]*; extra == '([^']+)'", requirement)
+                # Rewrite optional depdendencies:
+                # ruff; extra == "dev" -> ruff
+                match = re.match(r"([\S]+)[\s]*; extra == \"([^\"]+)\"", requirement)
```

Comment:
Shouldn't we just skip the dependency if `extra` is `dev`?

Reply:
I considered it but [optional dependencies](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html) appears to have no standard about group names within it. A project could calling it “linting” or “dev” or whatever and I didn’t want to add code to generic method `pipInstallSelective` that was unique to nnUnet. All optional dependencies will report as `package_name; extra == “group_name”`.

Alternatively the packagesToSkip could include know linting packages like black, ruff, pre-commit, however that list could arbitrarily grow and I didn’t want to start that precedent here either.